2019-06-11 01:58:48.823654: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-06-11 01:58:48.847488: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3691555000 Hz
2019-06-11 01:58:48.848118: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55a5f0f400b0 executing computations on platform Host. Devices:
2019-06-11 01:58:48.848181: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-06-11 01:58:48.984735: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55a5edc7d9a0 executing computations on platform CUDA. Devices:
2019-06-11 01:58:48.984793: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): GeForce GTX TITAN X, Compute Capability 5.2
2019-06-11 01:58:48.985525: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: GeForce GTX TITAN X major: 5 minor: 2 memoryClockRate(GHz): 1.076
pciBusID: 0000:03:00.0
totalMemory: 11.93GiB freeMemory: 11.81GiB
2019-06-11 01:58:48.985560: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-06-11 01:58:48.987876: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-06-11 01:58:48.987911: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-06-11 01:58:48.987927: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-06-11 01:58:48.988475: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11489 MB memory) -> physical GPU (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:03:00.0, compute capability: 5.2)
2019-06-11 01:58:53.778753: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.9.0 locally
Using TensorFlow backend.
Traceback (most recent call last):
  File "experiments/cifar_experiment.py", line 101, in <module>
    linear_mmd_real_vs_adversarial = mmd_evaluation(x_test[:1000], x_test_adv)
  File "/vol/gpudata/ap8415/adversarial-robustness-toolbox/experiment_models/utils.py", line 68, in mmd_evaluation
    x_flat = x.reshape(x.shape[0], x[0].shape[0] * x[0].shape[1])
ValueError: cannot reshape array of size 3072000 into shape (1000,1024)
